Sources/XcodeGenKit/PBXProjGenerator.swift

```diff
@@ -802,7 +803,7 @@ public class PBXProjGenerator {
                 
                 // If package's reference is none and there is no specified package in localPackages,
                 // then ignore the package specified as dependency.
-                if packageReference == nil, !project.localPackages.keys.contains(dependency.reference) {
+                if packageReference == nil, !localPackageReferences.contains(dependency.reference) {
```

Comment:
Does this still work? Can you have just a local Package with a path, without having an actual remove and then a mirrored local package

Reply:
I think this will work as is. In the first place, this validation should always be `false` because of project validation at [`SpecValidation.swift`](https://github.com/yonaskolb/XcodeGen/blob/master/Sources/ProjectSpec/SpecValidation.swift).

https://github.com/yonaskolb/XcodeGen/blob/59ab2239227811f2607a4981c8d304219ca5f47d/Sources/ProjectSpec/SpecValidation.swift#L176-L178